packages/form/core/form-group.ts

```diff
@@ -1,18 +1,24 @@
 import { ControlConfig, FormControl } from './form-control';
 import ShortUniqueId from 'short-unique-id';
 
+export interface FormConfig {
+	name?: string;
+	validateOnLoad?: boolean;
+}
+
 export class FormGroup {
 	controls: FormControl[];
 	name?: string;
 	id?: string;
 
-	constructor(controls: ControlConfig[], name = '') {
+	constructor(controls: ControlConfig[], formConfig: FormConfig = {}) {
 		const uid = new ShortUniqueId({ length: 9 });
+		const { name = '', validateOnLoad = true } = formConfig;
```

Comment:
I'm now thinking it is better to be `false` by default, same as `showValidationHints`

Reply:
Should it be `false` by default in `FormControl` and `FormGroup` too?

Reply:
I'd say yes they should have the same default... but maybe you don't have to set it, because `undefined` is also falsy?

Reply:
Yeah, but for type safety I should set it to `false`. 😄